sdk/python/tests/unit/test_feature_views.py

```diff
@@ -2,12 +2,14 @@
 
 import pytest
 
-from feast import PushSource
+from feast import Entity, Field, PushSource
```

Comment:
nit: we want to do `from feast.entity import Entity` etc. for all internal files